src/Components/Artwork/GridItem.tsx

```diff
@@ -44,37 +43,52 @@ class ArtworkGridItemContainer extends React.Component<Props, State> {
   private image: HTMLImageElement = null
 
   state = {
-    width: 1,
-    height: 1,
+    isMounted: false,
   }
 
   componentDidMount() {
-    const scale = window.devicePixelRatio
-    const width = this.image.width * scale
-    const height =
-      (this.image.width / this.props.artwork.image.aspect_ratio) * scale
-
     this.setState({
-      width,
-      height,
+      isMounted: true,
     })
   }
 
-  get imageURL() {
+  getImageUrl(breakpoints) {
     const imageURL = this.props.artwork.image.url
-    if (imageURL) {
+
+    if (!imageURL) {
+      return null
+    }
+
+    if (this.state.isMounted) {
+      const [[match]] = Object.entries(breakpoints).filter(([key, val]) => val)
+
+      const getWidth = () => {
+        switch (match) {
+          case "xs":
+          case "sm":
+          case "md":
+            return 400
+          case "lg":
+          case "xl":
+          default:
+            return 300
+        }
+      }
+
+      const scale = window.devicePixelRatio
```

Comment:
can we drop the pixel ratio thing for now? it can result in a resize request of 800px (the 400px case * 2). The large images themselves are only 640px max so this is actually worse, plus upscales the image.

i dont really notice a difference / am unfamiliar with this setting?

Reply:
The pixel ratio defines the number of pixels that make up a “point”. A “retina” screen has twice the amount of pixels per axis.

I guess if images get scaled up you won’t notice any difference indeed, so that requires a little more thinking, but I wouldn’t throw out the baby with the water just yet. Often devices with larger pixel ratios have better connections. We’re also all about showing images and the difference _should_ be noticeable. In my opinion we should talk to Brian.

Reply:
We should, but for now I'm comfortable with dropping it. This whole approach probably deserves some thinking (and thus prioritization and time allocation within our sprints), but for now, this basically matches existing Force behavior, which should obviously be fine to roll with (for now!).

Reply:
Sounds good 👍 (Be sure to file a ticket.)